src/components/Editor/Invitees/InviteesList.vue

```diff
@@ -161,6 +191,30 @@ export default {
 				attendee,
 			})
 		},
+		async createTalkRoom() {
+			try {
+				this.creatingTalkRoom = true
+				const url = await createTalkRoom(this.calendarObjectInstance.title)
+
+				let newDescription
+				if (!this.calendarObjectInstance.description) {
+					newDescription = url
```

Comment:
Add a new line at the end so it's easier to edit afterwards.

Reply:
Resolved